react/features/conference/components/native/carmode/styles.js

```diff
@@ -0,0 +1,156 @@
+import { fixAndroidViewClipping } from '../../../../base/styles';
```

Comment:
Have you tested this? it was necessary when drawing video with elements on top, which doesn't happen in car mode, so I think we don't need it here.